server/tests/src/test/java/org/infinispan/server/security/AuthenticationLDAPIT.java

```diff
@@ -57,8 +57,8 @@ public void testReadWrite() {
                .saslMechanism(mechanism)
                .serverName("infinispan")
                .realm("default")
-               .username("admin")
-               .password("strongPassword");
+               .username("admin_user")
```

Comment:
`javax.security.sasl.SaslException: ELY05161: Unable to retrieve password for "admin_user"`
test is failing